src/stp/plays/referee_specific/FreeKickThem.cpp

```diff
@@ -2,31 +2,76 @@
 // Created by jordi on 07-05-20.
 //
 
-#include "stp/plays/referee_specific/FreeKickThem.h"
-#include "stp/invariants/game_states/FreeKickThemGameStateEvaluation.h"
+#include "include/roboteam_ai/stp/plays/referee_specific/FreeKickThem.h"
 #include "include/roboteam_ai/stp/roles/passive/Defender.h"
+#include "include/roboteam_ai/stp/computations/PositionComputations.h"
+#include "include/roboteam_ai/stp/roles/Keeper.h"
 #include "include/roboteam_ai/stp/roles/passive/Formation.h"
-#include "stp/roles/Keeper.h"
 
 namespace rtt::ai::stp::play {
+const char* FreeKickThem::getName() { return "FreeKickThem Template"; }
```

Comment:
Why change the name? No need to change the previous getName() at the bottom of the class